circle.yml

```diff
@@ -6,8 +10,31 @@ dependencies:
     - echo y | android update sdk --no-ui -a --filter extra-google-google_play_services
     - echo y | android update sdk --no-ui -a --filter extra-google-m2repository
     - echo y | android update sdk --no-ui -a --filter extra-android-m2repository
+    - echo y | android update sdk --no-ui --all --filter sys-img-armeabi-v7a-google_apis-24
```

Comment:
I think you should use "-a" instead of "--all" just to keep consistency. Or maybe change all to "--all".